.github/workflows/build-and-test.yaml

```
@@ -36,7 +36,14 @@ jobs:
           target-packages: ${{ steps.get-self-packages.outputs.self-packages }}
           build-depends-repos: build_depends.repos
 
+      - name: Check file existence
+        id: check_files
+        uses: andstor/file-existence-action@v1
+        with:
+          files: lcov/total_coverage.info,coveragepy/.coverage
+
       - name: Upload coverage to Codecov
+        if: steps.check_files.outputs.files_exists == 'true'
```

To make the style consistent.

```suggestion
        if: ${{ steps.check_files.outputs.files_exists == 'true' }}
```